src/main.rs

```diff
@@ -1,43 +1,41 @@
-use actix_web::{get, App, HttpResponse, HttpServer, Responder};
-use sqlx::{Pool, Postgres};
+use actix_web::{web, App, HttpResponse, HttpServer, Responder};
+use anyhow::Result;
+use dotenv::dotenv;
+use sqlx::postgres::PgPool;
 use std::env;
-use std::io::ErrorKind;
 
-#[get("/")]
-async fn hello() -> impl Responder {
-    HttpResponse::Ok().body("Hello world!")
+// import user module (routes and model)
+mod user;
+
+// / handler
+async fn index() -> impl Responder {
+    HttpResponse::Ok().body(r#"
+        Available routes:
+        GET /users/ -> list of all users
+        POST /user/ -> create new user, example: { "id": 1, "first_name": "Shungo", "last_name": "Najima" }
+        GET /user/{id}/ -> show one user with requested id
+        PUT /user/{id}/ -> update user with requested id, example: { "id": 1, "first_name": "Sarasa", "last_name": "Najima" }
+        DELETE /user/{id} -> delete user with requested id
```

Comment:
these should make these all `/users/` -- saves frontend time from the confusion anyway and it's just standard to do that i believe